packages/nc-gui-v2/components/smartsheet/Gallery.vue

```diff
@@ -12,235 +13,86 @@ const isPublicView = false
-              </v-carousel> -->
-
-          <!--            <v-card-title -->
-          <!--              class="text-capitalize" -->
-          <!--              v-text="row[primaryValueColumn]" -->
-          <!--            /> -->
-          <v-card-text>
-            <v-container fluid>
-              <!--              <v-row class="">
-                    <v-col v-for="col in meta.columns" :key="col.title"
-                           class="col-12"> -->
-              <div v-for="col in meta.columns" :key="col.title" class="my-8">
-                <!--                <v-col v-for="col in fields" v-show="showFields[col.title]" :key="col.title" class="col-12 mt-1 mb-2"> -->
-                <!--
-    todo:header cell
-    -->
-                <label :for="`data-table-form-${col.title}`" class="body-2 text-capitalize caption grey--text">
-                  <SmartsheetHeaderVirtualCell v-if="isVirtualCol(col)" :column="col" />
-                  <SmartsheetHeaderCell v-else :column="col" />
-                  <!--                    <virtual-header-cell
-                          v-if="isVirtualCol(col)"
-                          :column="col"
-                          :nodes="nodes"
-                          :is-form="true"
-                          :meta="meta"
-                        />
-                        <header-cell
-                          v-else
-                          :is-form="true"
-                          :value="col.title"
-                          :column="col"
-                        /> -->
-                </label>
-                <div class="mt-2">
-                  <SmartsheetVirtualCell v-if="isVirtualCol(col)" v-model="row[col.title]" :edit-enabled="false" :column="col" />
+  return Array.isArray(val) && val.length === 0
+}
+</script>
 
-                  <SmartsheetCell
-                    v-else
-                    v-model="row[col.title]"
-                    :edit-enabled="false"
-                    :column="col"
-                    @update:model-value="updateRowProperty(row, col.title)"
-                  />
-                </div>
-                <!--                  <virtual-cell
-                        v-if="isVirtualCol(col)"
-                        ref="virtual"
-                        :column="col"
-                        :row="row"
-                        :nodes="nodes"
-                        :meta="meta"
-                      />
-                      <table-cell
-                        v-else
-                        :value="row[col.title]"
-                        :column="col"
-                        :sql-ui="sqlUi"
-                        :is-locked="isLocked"
-                        class="xc-input body-2"
-                        :meta="meta"
-                      /> -->
-                <!--                </v-col> -->
-                <!--              </v-row> -->
+<!-- TODO: Fix scrolling -->
+<template>
+  <div class="flex flex-col h-100 min-h-0 w-100">
```

Comment:
don't use `h-100` or `w-100` - those are vuetify styles.
Use `h-full` and `w-full` - TailwindCSS classes.